src/pset/map/output.rs

```diff
@@ -117,13 +165,46 @@ impl Output{
         rv
     }
 
+    /// Create a output from unblinded txout
+    /// This is to be used when the txout is not blinded. This sets
+    /// the blinding key from the txout nonce
+    pub fn from_txout(txout: TxOut) -> Self {
```

Comment:
in `PartiallySignedTransaction::from_tx`
```
let outputs = tx.output.into_iter().map(Output::from_txout).collect();
```
Should be changed to use `from_blinded_txout` if the output is blinded.

Reply:
something along the line of ecfe679b2db625e628515aa95a0c54d1f750d6f5. 
Maybe we can also remove `from_blinded_txout` and do the `if output.is_partially_blinded()` inside `from_txout`

Reply:
Agreed in principle. Though note that this will create a diversion from how elements `createpsbt` and `converttopsbt` work. 
```
if (txout.nNonce.IsCommitment()) {
    output.m_blinding_pubkey.Set(txout.nNonce.vchCommitment.begin(), txout.nNonce.vchCommitment.end());
}
```

The above RPCs don't check any status of blinding of the transaction and assume the transaction to be unblinded. I am okay(and in support of) with changing this here but highlighting that this is different from elements core RPC.

Reply:
Did something similar in the last commit